_pages/pre-work.md

```diff
@@ -2,15 +2,164 @@
-    - TL should ask when assigning pre-work what language they'll use so we can match to an interviewer with some conversant skill there
-- Review of pre-work should focus on "why did you", "how did you", etc - not technical gotchas
-- Pre-work is "due" 1 biz day before the interview, to give time for the reviewer to review
\ No newline at end of file
+**Goal: bring technical work to an interview. Discuss your code, thought processes, and choices with an interviewer.**
+
+Candidates have two options: complete a coding exercise, or bringing a
+code sample (see the [instructions to candidates](#instructions-to-candidates)
+below for the nitty gritty details).
+
+As an interviewer, you'll receive that code at least a day before your interview
+with the candidate. You should plan to spend an hour or two reading the
+candidate's code, running it, and preparing follow-up questions to ask when you
+interview them.
+
+## Before the interview
+
+Before the interview, prepare by reviewing the code, trying to run it (and the
+tests), and thinking about questions you'll want to ask the candidate during
+the interview (see below).
+
+As you review, remember that candidate only spent about 3 hours on this code;
+it doesn't need to be perfect. It should work, and solve the problem assigned,
+but minor issues, rough edges, and less-than-perfect parts are totally fine.
+A good way to think about this: you're looking for code that's about the
+same level as you might expect on an initial pull request. It might not be
+perfect yet, but that's what code review is for.
+
+For the list of things to look for during code review, see: [what to look for during code review](https://docs.google.com/document/d/12q9DsSSdqV388M6DOncun28ZDRT-B6wXrsrG5PbTsAA/edit#heading=h.t0hley37jiee)
+
+## The Code Review Interview
+
+After reviewing the code, you'll hold your interview with the candidate.
+The focus of the interview should be on their code, how it works,
+why they made the choices they did, and so forth.
+
+When interviewing, please [take notes]({{ site.baseurl }}/interviews/#take-notes),
+and *note what the candidate says, rather than your impressions* — that will
+help you share behavior reasons for your conclusions and decisions.
+
+**Remember to be as pleasant and friendly as you can be!** You can deliver
+a demanding interview while being kind and empathetic.
+
+[For more information on interviewing in general, check out the interviewing guide]({{ site.baseurl }}/interviews/).
+
+### Introductory Statement
+
+**You should say this, or something close to it, before beginning:**
+
+> *Thanks for interviewing with me today. This is a code review interview, which
+means I’ll ask a series of questions about the code sample you submitted, how it
+works, why you made the choices you did, and so on. There are no “right”
+answers; I’m interested in talking through your code with you. I’ve got about
+4-5 questions, and this will take us about 30 minutes.*
```

Comment:
It's just a stylistic thing, but I wonder if we need the italics AND the block quote styles here. I think I'd vote for dropping italics and just having the block quote.

But I'm also fine with leaving it as is. Just thinking out loud.